Tests/scripts/server_get_artifact.sh

```diff
@@ -18,13 +18,13 @@ if [[ "$ARTIFACT_BUILD_NUM" = "" ]]; then
     exit 1
 fi
 
-SERVER_DOWNLOAD_LINK=$(curl -s -H "$ACCEPT_TYPE" ${SERVER_API_URI}/${ARTIFACT_BUILD_NUM}/artifacts?${TOKEN_ATTR} | jq '.[].url' -r | grep demistoserver | grep /0/)
```

Comment:
what are the purpose of this change?

Reply:
Cause of @ronykoz changes in another branch and I don't want to merge from master

Reply:
server installer download returned more then one result with those properties (for some reason) in build 32322

Reply:
![image](https://user-images.githubusercontent.com/10514677/47026592-7ab39f80-d16e-11e8-91e0-4ce956230def.png)

jus saw this revert
cool